src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.file;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String filePath) {
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new java.io.FileWriter(filePath, false))) {
+            bufferedWriter.write(resultingReport);
+            bufferedWriter.newLine();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
```

Comment:
Throwing a `RuntimeException` for an `IOException` might not be the best approach as it can terminate the application abruptly. Consider using a custom exception or handling the error more gracefully to provide better feedback to the user.